app/scripts/src/content/content-controller.js

```diff
@@ -31,8 +31,8 @@ ContentController.prototype = {
   },
 
   onSearchError: function(status, response) {
-    this.sendAnalyticsEvent({ name: 'onSearchError', value: status });
-    console.error('traktflix: Search error', status, response);
+    this.sendAnalyticsEvent({ name: 'onSearchError', value: (this.item && this.item.title) ? this.item.title : status });
+    console.error('traktflix: Search error', status, response, this.item.title);
```

Comment:
I can't really see how it could perform a search without the title (it should not be able to get the url), but if it is the case this would result in a ReferenceError. For a debugging perspective it would still point to where you are.